### PR TITLE
restructuring validators and adding some new content

### PR DESCRIPTION
**Pull request by schimih**
#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [ ] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [x] other

### CODE REVIEW

File: docs/learn/EGLD.md [hunk 1]
```diff
@@ -3,14 +3,50 @@ id: EGLD
...
-7. **Fast finality for cross-shard transactions** in mere seconds. Having a very high TPS is required for a high throughput blockchain solution, but TPS is only half the picture: fast finality for cross-shard transactions is of crucial importance. Most existing state-of-the-art blockchain architectures refuse to mention this aspect, but from a user standpoint it is extremely important. Fast cross-shard finality is naturally handled by MultiversX at the protocol level, using a dispatching algorithm and a routing protocol.
+- **Ownership:** EGLD allows you to be your own bank, with full control over your funds.
+- **Security:** Protected by cryptography, exists under your own account, ensuring the safety of your funds and transactions.
+- **Decentralized:** No central authority controls EGLD, making it a truly decentralized currency.
+- **Accessibility:** All you need is an internet connection and a wallet to use EGLD. Forget about banks.
+- **Flexibility:** EGLD can be divided into small units(as small as 0.000000000000000001), allowing for transactions of any size.
```
Comment (bogdan-rosianu) [inline docs/learn/EGLD.md:14]: add a space after `units`

Comment (bogdan-rosianu) [reply docs/learn/EGLD.md:14]: also, add empty line before?

**Review by miiu96 — APPROVED** [review]

File: docs/learn/multiversx-ecosystem.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+:::
+
+[comment]: # (mx-context-auto)
+
+## **What is MultiversX?**
+
+MultiversX is a distributed transactional computation protocol that relies on a sharded state architecture and a [Secure Proof of Stake](/docs/learn/consensus.md) consensus mechanism. While most other blockchain networks require custom hardware and high energy consumption, MultiversX runs on [average computers](/docs/validators/system-requirements.md).
+
+By employing [_sharding_](/docs/lea), a method of parallelizing data & transactions processing, MultiversX’s performance will scale up with the number of computers joining the network, reaching more than 100.000 transactions per second while growing increasingly decentralized.
```
Comment (bogdan-rosianu) [inline docs/learn/multiversx-ecosystem.md:24]: 100.000 => 100,000

File: docs/learn/multiversx-ecosystem.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+:::important
+**What is a blockchain?**
+
+A blockchain is a database of transactions that is continually updated and shared across numerous computers in a network. Each new set of transactions is grouped into a "block," which is added to the chain of previous transactions, hence the name blockchain. Public blockchains like MultiversX enable anyone to add data, but not remove it (**immutable**). To alter any information or manipulate the system, one would need to compromise the majority of computers on the network, which is an incredibly challenging task. This inherent difficulty makes decentralized blockchains like MultiversX highly secure and trustworthy.
```
Comment (bogdan-rosianu) [inline docs/learn/multiversx-ecosystem.md:29]: "which is added to the chain of previous transactions" -> maybe "previous blocks"?

File: docs/learn/multiversx-ecosystem.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+By solving some of the hardest consensus and sharding problems in the blockchain space, MultiversX is able to provide a very high level of performance on a network made of inexpensive computers, resulting in a very low cost per transaction. In addition to performance and cost, MultiversX also stands out through the quality of the developer experience and the resulting boost in usability on the end-user side.
+
+[comment]: # (mx-context-auto)
+
+## **Why would I use MultiversX?**
+
+MultiversX is a collaborative story written by all of us, inviting any builder to discover the digital universes that can be built together no matter that it is in DeFi, Gaming or Payments area. MultiversX nodes are distributed to more than 125 different staking providers, ensuring a robust and secure network. It becomes invaluable for individuals facing uncertainty regarding the security, stability, or mobility of their assets due to external forces beyond their control, and thanks to the ESDT standard and guardians that enhance its security.
```
Comment (bogdan-rosianu) [inline docs/learn/multiversx-ecosystem.md:46]: "it is in DeFi, Gaming or Payments area. MultiversX nodes are distributed to more than 125 different staking providers"
maybe we should also mention (maybe with a count) the number of individual nodes/identities that are not SPs

File: docs/learn/multiversx-ecosystem.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+**Important achievements:**
+
+- First live blockchain architecture with state sharding
+- **MainChain: **+30,000 TPS(scales > 100kTPS), 6s latency, $0.002/tx
+- **Sovereign Chain: **+78,000 TPS, 1s latency
+- xPortal App: buy, store, earn, pay EGLD, BTC, ETH, BNB
+- Smart Contracts, Staking & Delegation, Tokens
+- Validated through multiple audits from Trial of Bits, RuntimeVerification, Certik & others
```
Comment (bogdan-rosianu) [inline docs/learn/multiversx-ecosystem.md:57]: Trail of Bits

**Review by miiu96 — APPROVED** [review]

File: docs/learn/getting-started.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
+---
+id: getting-started
+title: Getting Started
+---
+
+MultiversX is a high-performance, sharded smart contract blockchain with Proof of Stake (SPoS) consensus, perfectly designed for the Web3 era with capacity to serve Web2 needs. At the moment of writing, it achieves over 30,000 TPS with adaptive scalability, offering fast and low-cost transactions. The xPortal money app, with embedded DeFi and cross-chain operations, empowers users to interact and own their digital assets. Developers benefit from integrated tools and 30% gas royalties, while validators earn ~8% APR. MultiversX offers sovereign chains for flexible, interoperable blockchain solutions, serving enterprise needs, driving the Web3 vision forward. It is a public blockchain similar to Ethereum, users own their accounts and tokens and they are free to move between [more than 500 different apps](https://multiversx.com/ecosystem).
```
Comment (bogdan-rosianu) [inline docs/learn/getting-started.md:6]: Proof of Stake (SPoS) => Secure Proof of Stake (SPoS)

**Review by miiu96 — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**